kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java

```diff
@@ -684,6 +685,60 @@ public void testUserTaskSetTaskProperties() throws Exception {
         }
     }
 
+    /**
+     * Test verifying client methods concerning task comments - addTaskComment(), getTaskCommentById(), getTaskCommentsByTaskId().
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testUserTaskComments() throws Exception {
+        assertSuccess(client.createContainer("definition-project", new KieContainerResource("definition-project", releaseId)));
+
+        Long processInstanceId = processClient.startProcess("definition-project", "definition-project.usertask");
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+        try {
+            List<TaskSummary> taskList = taskClient.findTasksAssignedAsPotentialOwner("yoda", 0, 10);
```

Comment:
@triceo Can do, some things can be turned into test suite wide constants, others just local ones.